### PR TITLE
DBW: Add helpText to report

### PR DESCRIPTION
**Pull request by ebidel**
R: @paulirish @brendankenny @GoogleChrome/lighthouse 

Not worth writing tests at this point. Just need something that's less confusing than the current double negatives for failure results: "Site is not using AppCache: No". This also adds help text for each feature and suggestions (with links) on what to use instead.

Current:

screen shot 2016-09-22 at 6 13 54 pm

Now:

screen shot 2016-09-22 at 6 05 28 pm

Eventually we should add a custom report instead of style overrides.

### CODE REVIEW

**Comment by paulirish** [comment]
IMO we should just change this in report generator. I think this is superior to what the default report has. (Green "Yes", red "No")

**Comment by ebidel** [comment]
Happy to do that if I have the 👍. Don't want to spend time on it and then be :(

**Comment by paulirish** [comment]
:+1: from me and @brendankenny on changing the default reporter to use check and X

**Comment by ebidel** [comment]
PTAL

screen shot 2016-09-22 at 9 25 28 pm

File: lighthouse-core/report/report-generator.js [hunk 1]
```diff
@@ -55,11 +55,11 @@ class ReportGenerator {
...
     Handlebars.registerHelper('version', _ => pkg.version);
 
-    // Helper for either saying 'yes' or 'no' for booleans, or simply returning the value if it's
-    // of any other type.
+    // Helper for either show an "✘" or "✔" booleans, or simply returning the
+    // value if it's of any other type.
     Handlebars.registerHelper('getItemValue', value => {
       if (typeof value === 'boolean') {
-        return value ? 'Yes' : 'No';
+        return value ? '&#10004;' : '&#10008;';
```
Comment (paulirish) [inline lighthouse-core/report/report-generator.js]: whoa, never seen this escape. 
don't wanna just pass as a literal?

**Comment by paulirish** [comment]
@ebidel with https://github.com/GoogleChrome/lighthouse/pull/695/commits/2ec6b5a9006c6f6402f30b27b32396f11fbafc54 in place, do you need the formatters anymore? You can nuke those, right?

File: lighthouse-core/report/report-generator.js (hunk 1 above)
Comment (ebidel) [inline lighthouse-core/report/report-generator.js]: Gotta love up on those unicodes.

**Comment by ebidel** [comment]
Or maybe I don't? Is there a formatter that already does that?

**Comment by ebidel** [comment]
upstreamed the `helpText` to metadata:

screen shot 2016-09-23 at 9 20 31 am

**Comment by brendankenny** [comment]
I'll take a look at this this morning

**Comment by brendankenny** [comment]
sorry, not the morning anymore :)

**Comment by paulirish** [comment]
This is a great PR. Already loving the new looks.

**Pull request merged**